### PR TITLE
doe and debug class was added

### PR DESCRIPTION
**Pull request by pelegsch666**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ module.exports = {
                 '2xl': '1400px',
             },
         },
+
```
Comment (galeindor) [inline tailwind.config.js]: remove redundent space

File: src/styles/globals.css [hunk 2]
```diff
@@ -7,3 +7,7 @@ html,
 body {
 	height: 100%;
 }
+.debug{
```
Comment (galeindor) [inline src/styles/globals.css:10]: is this class necessary? what's wrong with writing the tailwind in the relevant place?

**Review by galeindor — CHANGES_REQUESTED** [review]
good work , 
few small points in design :
1. in dark mode the card in transparent , in my opinion it should have some background and drop shadow the same as in light mode
2. card title text isn't visiable in dark mode 
3. other notes mentioned in review

File: src/styles/globals.css (hunk 2 above)
Comment (pelegsch666) [reply src/styles/globals.css:10]: it is just for checkings it will go when we dont need it

Comment (galeindor) [reply src/styles/globals.css:10]: Ok ,understandable

**Pull request merged**